### ISSUE TITLE
[BUG] Inifite zoom with no option to zoom out in flatpak version

### ISSUE THREAD

**Issue by kreativK** [Milkshiift/GoofCord#42]
### Description
Pressing ctrl+shift and the two buttons to the left of backspace zooms out, but there is no way to zoom back in again. Reinstalling the app doesn't work, neither does ctrl+0 or similar shortcuts. I had to reinstall some flatpak libraries to fix it, and I dont know which one did.

### Steps to Reproduce
Run Goofcord and press ctrl+shift+one of the buttons to the left of backspace

### Expected Behavior
There is a way to zoom in again

### Actual Behavior
There isn't

### Environment
- Operating System (name and version): Fedora 40 Kinoite
- GoofCord Version: 1.5.1

**Comment by Milkshiift** [comment]
There is a way to zoom in, FAQ on the wiki answers your question https://github.com/Milkshiift/GoofCord/wiki/FAQ
What keyboard layout do you have?

**Comment by kreativK** [comment]
> There is a way to zoom in, FAQ on the wiki answers your question https://github.com/Milkshiift/GoofCord/wiki/FAQ What keyboard layout do you have?

I use swedish keyboard layout. Ctrl+/- didn't work and neither does ctrl+0.

**Comment by kreativK** [comment]
I can reproduce this in a virtual machine. Nothing in the wiki you linked worked. Disabling custom titlebar and using alt and zoom in/out only zooms in/out the menu, not the goofcord client. Switching to english keyboard layout doesn't change anything, ctrl+/- still doesn't do anything.

**Comment by Milkshiift** [comment]
This is an Electron issue much more than a GoofCord issue. Do you experience the same issue in other electron apps? VSCode, BitWarden, Figma, Notion, Slack, Signal for example

**Comment by kreativK** [comment]
Could not reproduce on Signal, Bitwarden, VSCode, Figma or Obsidian. Could reproduce on Slack but I could also zoom out again with ctrl+- and ctrl+0, which I can't in Goofcord.

**Comment by Milkshiift** [comment]
You mentioned that using alt menu options to zoom only affects the menu itself. I can't reproduce that, the options only scale Discord and don't affect the menu in any way. Do zoom keyboard [shortcuts](https://github.com/Milkshiift/GoofCord/blob/main/src/menu.ts#L83) affect the menu scaling? Because if alt menu options don't scale Discord, shortcuts won't either. They are executing the same code.
You should also try running GoofCord from an AppImage build

**Comment by kreativK** [comment]
> You mentioned that using alt menu options to zoom only affects the menu itself. I can't reproduce that, the options only scale Discord and don't affect the menu in any way. Do zoom keyboard [shortcuts](https://github.com/Milkshiift/GoofCord/blob/main/src/menu.ts#L83) affect the menu scaling?

No, they don't affect menu scaling. The only way to zoom out in the menu is with the button that comes up when pressing alt.

> You should also try running GoofCord from an AppImage build

Tried the AppImage build, no difference at all.

Try it in a vm with Fedora Kinoite, you will be able to reproduce it
